geniza/corpus/templatetags/corpus_extras.py

```diff
@@ -93,9 +93,20 @@ def iiif_image(img, args):
 
 
 @register.filter
-def iiif_info_json(image_ids):
+def iiif_info_json(images):
     """Add /info.json to a list of IIIF image IDs and dump to JSON,
     for OpenSeaDragon to parse. Example use::
 
     """
-    return json.dumps([image_id + "/info.json" for image_id in image_ids])
+    return json.dumps([image["image"].info() for image in images])
+
+
+@register.filter
+def format_attribution(attribution):
+    (attribution, additional_restrictions, extra_attrs_set) = attribution
+    extra_attrs = "\n".join("<p>%s</p>" % attr for attr in extra_attrs_set)
+    return '<div class="attribution"><p>%s</p><p>%s</p>%s</div>' % (
+        attribution,
+        additional_restrictions,
+        extra_attrs,
+    )
```

Comment:
this is awfully specific for a template tag; any reason this couldn't just be done in the template?

Reply:
I thought it might be nice to have that formatting function available to the ManifestView as well, but not sure what we're doing with that atm, so maybe it makes sense to just move it into the template.